specs/pvs/corla.pvs

```diff
@@ -438,8 +472,10 @@ BEGIN
 
 % @description A decision to be made, either the selection of one or
 % more candidates for an office or the choice of a course of action.
+% @todo kiniry Need to clarify choices vs options vs votes.
   ballot_contest: TYPE = contest
-  choice: TYPE
+  option: TYPE = string
+  choice: TYPE = set[option]
 
 % @description A batch is a set of ballots.  
   batch: TYPE = set[ballot]
```

Comment:
line 452/488 meaning of "imprinted ballot" should be made clear in prose. I can't do it since I don't know what "imprinted ballot" means.